docs/README.md

```diff
@@ -12,7 +12,7 @@
 
 For a quick guide on getting started, see the top-level [README](../README.md#getting-started).
 
-## Installation
+## Installation and basic usage
```

Comment:
Sorry to nitpick, but I'd rather leave this as just "Installation" and have a separate "Usage" or "Running ty" section. Is there a reason you'd prefer to combine them?

Reply:
Right now, the section in this document on running ty is only a few sentences, so I thought it was perfectly fine to combine it in this section for now. Sounds like you have a strong preference to instead have it be a top-level header; do you intend to add any more guidance for users in that section, or have all of that guidance be in `cli.md`?

Reply:
Changed.